misc/config_tools/acpi_gen/asl_gen.py

```diff
@@ -446,8 +446,9 @@ def main(args):
         PASSTHROUGH_PTCT = False
 
     kern_args = common.get_leaf_tag_map(scenario, "os_config", "bootargs")
+    vm_type = common.get_leaf_tag_map(scenario, "vm_type")
     for vm_id, passthru_devices in dict_passthru_devices.items():
-        if kern_args[int(vm_id)].find('reboot=acpi') == -1:
+        if kern_args[int(vm_id)].find('reboot=acpi') == -1 and vm_type[int(vm_id)] not in ['SAFETY_VM']:
```

Comment:
Wouldn't it be better to test for `KERNEL_ZEPHYR` here instead?

Zephyr does not take command-line arguments regardless of whether it runs as a safety VM or not.

A different question: assume this safety VM runs Linux or some other Safety OS that can accept such parameter, what should we do with it?